test/unittest.html

```diff
@@ -10,6 +10,11 @@
         <div id="messages"></div>
         <div id="fixtures"></div>
 
+        <script src="browser/stub-xhr.js"></script>
+        <script src="../node_modules/fetch-mock/es5/client-bundle.js"></script>
+        <script>
+          fetchMock.mock('*', 200);
```

Comment:
will this affect all fetch requests in all existing and new tests in future ?, if so I think this should be moved to certain unit test and be called in "beforeEach" block for only desired tests

Reply:
Good call.

Reply:
So because of how the plugins grab fetch/xhr on load the mocks need to be setup before the plugin is loaded. I've added much more specific urls, to prevent clashes between tests, and a comment to make it more clear what's going on and why.

Reply:
I would argue as I can  see here for example 
http://www.wheresrhys.co.uk/fetch-mock/
where it works exactly this way, it includes the "plugin" then requires fetchMock, then mock it and then reverting afterwards (reset).